packages/devtools_server/lib/src/handlers.dart

```diff
@@ -77,9 +82,19 @@ class ServerApi {
   FutureOr<shelf.Response> logScreenView(shelf.Request request) =>
       notImplemented(request);
 
+  /// Gets a user's LDAP for use in analytics and for distinguishing internal
```

Comment:
Be clear that this API will only be implemented for the version of DevTools used inside of Google.

Reply:
updated comment for clarity.